javalib/src/main/scala/java/util/logging/StreamHandler.scala

```diff
@@ -0,0 +1,88 @@
+package java.util.logging
+
+import java.io.OutputStream
+
+class StreamHandler(private[this] val _out: OutputStream,
+    private[this] val _formatter: Formatter) extends Handler {
+
+  // Required by javadoc but it is unspecified what to do if formatter is null
+  def this() = this(null, null)
+
+  protected var out:Option[OutputStream] = Option(_out)
```

Comment:
This should not be `protected`, it is not in the documentation. Consider not boxing the `_out`.